lib/boxr/auth.rb

```diff
@@ -53,6 +55,33 @@ def self.revoke_tokens(token, client_id: ENV['BOX_CLIENT_ID'], client_secret: EN
     auth_post(uri, body)
   end
 
+  # Exchange a fully-scoped token for one that is down-scoped to specific permissions,
```

Comment:
Would you mind greatly reducing the size of this comment? This codebase in general doesn't rely on comments too much since the API is supposed to match the library 1-to-1. On the other hand, I understand this method has a certain complexity in behavior worth being clear about. Maybe a one liner and what the token parameter should be?